tomb-common/src/banyan_api/models/storage_ticket.rs

```diff
@@ -204,16 +204,16 @@ pub mod test {
         let mut fs_metadata = FsMetadata::init(&key)
             .await
             .expect("Failed to create fs metadata");
-        let mkdir_path_segments = vec!["test".to_string(), "path".to_string()];
-        let add_path_segments = vec!["test".to_string(), "path".to_string(), "file".to_string()];
+        let mkdir_path_segments = &vec!["test".to_string(), "path".to_string()];
+        let add_path_segments = &vec!["test".to_string(), "path".to_string(), "file".to_string()];
```

Comment:
Shouldn't take references of concrete objects to be stored, where they're getting passed around is where you want to be taking references.